wps_shared/wps_shared/db/crud/auto_spatial_advisory.py

```diff
@@ -579,6 +592,54 @@ async def save_run_parameters(
     await session.execute(stmt)
 
 
+async def check_run_parameters_id_exists_in_all(
+    session: AsyncSession, run_parameters_id: int
+) -> bool:
+    """
+    Returns True if run_parameters_id exists in all required tables, False otherwise.
+    """
+    tables = [
+        HighHfiArea,
+        AdvisoryFuelStats,
+        AdvisoryTPIStats,
+        AdvisoryHFIWindSpeed,
+        AdvisoryHFIPercentConifer,
+        CriticalHours,
+    ]
+    for table in tables:
```

Comment:
Instead of this for loop, can we just join all tables on the `run_parameters_id` and make sure it exists?